platforms/web/public/locales/en/video.json

```
@@ -12,6 +12,11 @@
   "favorite": "Favorite",
   "favorites_warning": "Maximum amount of favorite videos exceeded. You can only add up to {{maxCount}} favorite videos. Please delete one and repeat the operation.",
   "live": "Live",
+  "player_error": {
+    "geo_blocked_description": "We apologize, but this content is currently unavailable in your region.",
+    "geo_blocked_title": "Content Unavailable in your region",
```

Perhaps a shorter title looks better on mobile? Also no cap `u`. :-) 
```suggestion
    "geo_blocked_title": "Content unavailable",
```

---

Changed!